devkit/sandbox/package.json

```diff
@@ -1,15 +1,27 @@
 {
-  "name": "@rpch/sandbox",
-  "version": "0.1.1",
-  "private": true,
-  "license": "LGPL-3.0",
-  "scripts": {
-    "build": "exit 0",
-    "dev": "exit 0",
-    "format": "exit 0",
-    "lint": "exit 0",
-    "test": "exit 0",
-    "start": "./src/start.sh",
-    "stop": "./src/stop.sh"
-  }
-}
\ No newline at end of file
+    "name": "@rpch/sandbox",
+    "version": "0.2.0",
+    "private": true,
+    "license": "LGPL-3.0",
+    "scripts": {
+        "prebuild": "node -p \"'export default \\'' + require('./package.json').version + '\\';'\" > src/version.ts",
+        "build": "yarn prebuild && tsc",
+        "dev": "exit 0",
+        "format": "prettier --write src/ .eslintrc.js package.json tsconfig.json",
+        "format:ci": "prettier --check src/ .eslintrc.js package.json tsconfig.json",
+        "lint": "exit 0",
+        "test": "exit 0",
+        "start": "./src/start.sh",
+        "stop": "./src/stop.sh"
+    },
+    "dependencies": {
+        "@hoprnet/hopr-sdk": "^2.0.1",
+        "dotenv": "^16.3.1",
+        "pg": "^8.11.3"
+    },
+    "devDependencies": {
+        "@types/dotenv": "^8.2.0",
+        "turbo": "^1.6.3",
```

Comment:
this shouldn't be part of the sandbox, but instead come from the mono repo root.

Reply:
Ok, changing